webrtc.html

```diff
@@ -248,36 +266,66 @@
               "idlMemberType"><a>DOMString</a></span></dt>
               <dd>
                 <p>If this <code><a>RTCIceServer</a></code> object represents a
-                TURN server, then this attribute specifies the username to use
-                with that TURN server.</p>
+                TURN server, then this attribute specifies the username in case
+                of password credential type, or the kid (The key ID of the 
+                symmetric key wich is used to encrypt/decrypt the access_token)
+                in case of token credential type to use with that TURN server.
+                </p>
               </dd>
               <dt><dfn><code>credential</code></dfn> of type <span class=
               "idlMemberType"><a>DOMString</a></span></dt>
               <dd>
                 <p>If this <code><a>RTCIceServer</a></code> object represents a
-                TURN server, then this attribute specifies the credential to
-                use with that TURN server.</p>
+                TURN server, then this attribute, in case <code>
+                credentialType="password"</code> then it specifies the password
+                / credential that to use with that TURN server, or elseif in
+                case <code>credentialType="token"</code> then it is the
+                key / mac_key that to use with that TURN server. As it is
+                described in [[!TRAM-TURN-THIRD-PARTY-AUTHZ]] Section 6.2. and
+                in Appendix B. Figure 8. "key"</p>
+              </dd>
+              <dt><dfn><code>accesstoken</code></dfn> of type <span class=
+              "idlMemberType"><a>DOMString</a></span></dt>
+              <dd>
+                <p>If this <code><a>RTCIceServer</a></code> object represents a
+                TURN server, then if <code>credentialType="token"</code> 
+                this attribute specifies the OAuth Access Token to use with 
+                that TURN server. The structure is described in
+                [[!TRAM-TURN-THIRD-PARTY-AUTHZ]] Section 6.2.
+                In case <code>credentialType="password"</code> then this field
+                is not used, and should have empty value.</p>
```

Comment:
This is a dictionary, and the member doesn't have a "required" keyword. So the correct behavior is to omit it, not to give it the empty value.